### PR TITLE
building component class and styling in scss, #10

### PR DESCRIPTION
**Pull request by szarlata**
JS: Creating the component as a class. 
SCSS: Partial styling of the button.

### CODE REVIEW

**Review by DaveTheWebDev — COMMENTED** [review]
You can write hall of fame string in one line, make separate parent section which will be container for all buttons and append your button to it. Remove _ from js filename. Consider changing i tag to span tag because span has no semantic value. BEM used very well, group properties in scss according to Przemek message https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/ 
Good PR with minior improvements :)

File: src/app/components/_buttonhall.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class ButtonHall {
+  render() {
+    const text = "Hall of fame";
+    const hallText = text.toUpperCase();
+    const hallButton = `<button class="buttonHall">
+                          <i class="buttonHall__icon"><img src="static\assets\ui\icon_contacts_24px.png"></i>
+                          <span class="buttonHall__text">${hallText}</span>
+                        </button>`;
+    document.getElementById("swquiz-app").innerHTML += hallButton;
+  }
```
Comment (ZibiH) [inline src/app/components/_buttonhall.js:10]: Hmm... this class can be used to create 'Hall of fame' button only. What you think about creating general class (e.g. ButtonCreator), then applying it to this particular button?

**Review by ZibiH — COMMENTED** [review]
2 comments / questions inside

**Comment by pspaczek** [comment]
@szarlata please add a title for PR and id of issues in description like `Related issues: #10`

**Comment by szarlata** [comment]
Outdated. Button class for both kind of buttons was created on branch 9-UI...

**Pull request closed without merging**